hosting/package.json

```diff
@@ -2,23 +2,23 @@
   "name": "codesupport-code-review-tool-frontend",
   "version": "0.1.0",
   "private": true,
+  "type": "module",
   "dependencies": {
     "@tanstack/react-query": "^5.34.1",
-    "@types/node": "^16.18.96",
+    "@types/node": "^20.12.11",
```

Comment:
I appreciate it wasn't already, but this should probably be moved to be a `devDependency`.

Reply:
Yes! I also moved the rest of the bunch and typescript down 👍